emojirades/analytics/time_range.py

```diff
@@ -4,15 +4,16 @@
 
 
 class TimeRange:
-    pendulum_unit = {
-        TimeUnit.WEEKLY: "week",
-        TimeUnit.MONTHLY: "month"
-    }
+    pendulum_unit = {TimeUnit.WEEKLY: "week", TimeUnit.MONTHLY: "month"}
```

Comment:
I prefer the previous version. But I guess we have to follow Black.

Reply:
Yeah black...